### ISSUE TITLE
Upgrade to 2.0.0 -> images are not processed anymore

### ISSUE THREAD

**Issue by tecbeast42** [1000ch/gulp-image#14]
After i upgraded to 2.0.0 my images are not processed anymore.

Output from ^1.0.0
```
[12:38:39] Starting 'imageOptimize'...
[12:38:39] Finished 'default' after 6.83 ms
[12:38:39] - Skyline_hintergrund.svg -> Cannot improve upon 2.66 KB
[12:38:39] - Skyline_hintergrund_ohneWolken.svg -> Cannot improve upon 2.16 KB
[12:38:39] - bullet-dark.svg -> Cannot improve upon 1.85 KB
[12:38:39] - bullet.svg -> Cannot improve upon 1.3 KB
[12:38:39] - clubhaus.svg -> Cannot improve upon 46.48 KB
[12:38:39] - dach.svg -> Cannot improve upon 2.62 KB
[12:38:39] - logo.svg -> Cannot improve upon 5.69 KB
[12:38:39] - siegel_geldzurueck.svg -> Cannot improve upon 10.1 KB
[12:38:39] - siegel_service.svg -> Cannot improve upon 8.72 KB
[12:38:39] - sprechblase.svg -> Cannot improve upon 1.13 KB
[12:38:39] - wolke1.svg -> Cannot improve upon 836 B
[12:38:39] - wolke2.svg -> Cannot improve upon 766 B
[12:38:39] - favicon/favicon.svg -> Cannot improve upon 2.25 KB
[12:38:39] ✔ favicon/favicon.png -> before=792 B after=565 B reduced=227 B(28.7%)
[12:38:39] ✔ siegel_geldzurueck.png -> before=14.53 KB after=6.86 KB reduced=7.67 KB(52.8%)
[12:38:39] ✔ kurve_boxen.png -> before=3.55 KB after=3.49 KB reduced=55 B(1.5%)
[12:38:39] Finished 'imageOptimize' after 561 ms
```
Output from ^2.0.0
```
[12:40:55] Starting 'default'...
[12:40:55] Starting 'imageOptimize'...
[12:40:55] Finished 'default' after 6.68 ms
```

**Comment by ubenzer** [comment]
I have the same issue, using node v5.11.0.

**Comment by andrewjmead** [comment]
Same issue here. Using 2.0.0 on Node v6.1.0.

**Comment by lucaslim** [comment]
Error seems to be with jpegoptim.

It's returning error if more than one options is set for jpeg optimization

~~~~
jpegoptim: error removing file: C:\Users\<User>\AppData\Local\Temp\84dd72bf-abe9-4a5d-b174-ee76d4b3f743\Sam_bike_high_res.jpg
jpegoptim: cannot rename temp file
~~~~

**Comment by 1000ch** [comment]
Hmm, do you show me your image optimisation task in `gulpfile.js`?

**Comment by ubenzer** [comment]
```js
      pipe = pipe.pipe(g.image({
        svgo: {disable: ["convertTransform"]}
      }));
```

**Comment by elbebass** [comment]
Just ran into this bug

**Comment by ecDusty** [comment]
Hi Team, I think I found a work around. (I'm also new to contributing so please bare with any small mistakes I might make)

I've been running into the same issue after I installed the updated version of gulp-image v2.0.1.  I had previously been using gulp-image v1.3.1 without issue. (Node v4.4.4, npm v3.8.6, Gulp v3.9.1)

When I tried to use gulp-image v2.0.1, it would crash out and not complete. My gulp code read:
```
var = IMG_PATH = 'src/**/*.{jpeg,png,jpg,gif,svg}';
var DIST_DIR = 'dist'; 

gulp.task('images', function () {
  console.log('>---Starting Images function---<');

  return gulp.src(IMG_PATH)
    .pipe(imgS())
    .pipe(gulp.dest(DIST_DIR));
});
```

Within my Git Bash console, the program would crash without any reason (See example below)
```
xxx@xxx-Yoga  /c/GitHub/mobile-portfolio-fend (master)
$ gulp images
[15:29:00] Using gulpfile C:\GitHub\mobile-portfolio-fend\gulpfile.js
[15:29:00] Starting 'images'...
>---Starting Images function---<
[15:29:00] - images\arcade-game.jpg -> Cannot improve upon 5.81 KB

xxx@xxx-Yoga  /c/GitHub/mobile-portfolio-fend (master)
$
```

1) Trying to keep as many options set to true:
It seems that if you disable certain image optimization features that gulp-image wouldn't crash. (Please note that I went through every possible configuration, trying to leave as many options set to true as possible). See my edited code that worked below:
```
gulp.task('images', function () {
  console.log('>---Starting Images function---<');

  return gulp.src(IMG_PATH)
    .pipe(imgS({
        pngquant: true,
        optipng: true,
        zopflipng: false,
        advpng: false,
        jpegRecompress: false,
        jpegoptim: true,
        mozjpeg: false,
        gifsicle: false,
        svgo: false
      }))
    .pipe(gulp.dest(DIST_DIR));
});
```

2) Testing each image optimization separately
To be vigilant in my testing, I tried setting each image optimization to true 1 at a time (Leaving the rest set to false). It seems that 3 options crash out by themselves:
 - jpegRecompress
 - jpegoptim
 - svgo


I do hope this helps. If you have any questions I'd be happy to help out.

**Comment by 1000ch** [comment]
@tecbeast42 @ubenzer @andrewjmead @lucaslim @elbebass @ecDusty 
Sorry for delay, this may be fixed on `v2.1.0`.
